docusaurus/src/remark/utils.js

```diff
@@ -1,29 +1,53 @@
 const { catalog } = require("../connector_registry");
 
+// the migration guide and troubleshooting guide are not connectors, but also not in a sub-folder, e.g. /integrations/sources/mssql-migrations
+const connectorPageAlternativeEndings = ["-migrations", "-troubleshooting"];
+const connectorPageAlternativeEndingsRegExp = new RegExp(
+  connectorPageAlternativeEndings.join("|"),
+  "gi"
+);
+
 const isDocsPage = (vfile) => {
+  let response = { isDocsPage: false, isTrueDocsPage: false };
```

Comment:
This is the main change - the page utils now need to report both isDocsPage (should get the header) AND isTrueDocsPage (should get the AirbyteLib inclusions)